lib/v1_alpha_fetcher.rb

```diff
@@ -0,0 +1,45 @@
+class V1AlphaFetcher
+  def self.fetch(url)
+    url += "/apis"
+
+    uri = URI(url)
+
+    req = Net::HTTP::Get.new(uri)
+    req["correlation-id"] = SecureRandom.uuid
+    req["accept"] = "application/vnd.uk.gov.api.v1alpha+json"
+
+    response = Net::HTTP.start(uri.hostname, uri.port, use_ssl: uri.scheme == "https") do |http|
+      http.request(req)
+    end
+    raise "Could not parse APIs from #{url}, no version supported" if response.is_a? Net::HTTPNotAcceptable
```

Comment:
did we want to raise a specific exception, allowing the caller to handle them more nicely?